lib/all_your_base_cases.rb

```diff
@@ -20,9 +20,7 @@ def skipped
   private
 
   def indent(size, lines)
-    lines.split("\n").map do |line|
-      ' ' * size + line
-    end.join("\n")
+    lines.lines.each_with_object('') { |line, obj| obj << ' ' * size + line }
```

Comment:
`lines.lines` looks weird.
If you need to call `lines` on it that suggests to me that it's not lines at all, so either needs to be lines that get passed in or needs a different name.